package.json

```diff
@@ -11,15 +11,15 @@
   },
```

Comment:
- [ ] **Required:** The start script is missing and even after supplying it on my cloned version I still have a loading error.

![image](https://user-images.githubusercontent.com/27709832/186440076-f602bbfe-7685-4747-b268-a060eca4e536.png)


Kindly include the script:
`"start": "webpack serve --open"`

This would require the npm installation below:
`'npm install -D webpack-dev-server'`

I will mark it as invalid on your dashboard. Kindly fix this and request another review.